### DISCUSSION TITLE
[New User] Syncing 16-color to Vim in Kitty

### DISCUSSION THREAD

**Discussion by wllclngn** [kovidgoyal/kitty#4287] [Q&A] [answered]
Just came to Kitty today from Alacritty. I've tried finding an answer to this, but cannot immediately find it. Apologies if the answer exists.

I have "t_Co=16" in *vimrc to use the colorscheme from Kitty's *conf while using Vim. However, many of the colors are system-default colors when using Vim.

I feel that I'm missing an option in my Kitty *conf file. But, being a new user, I'm unsure what that would be. Any help is greatly appreciated.

**Comment by kovidgoyal** [comment, marked as answer]
This has nothing to do with the terminal emulator. You need to convince
vim to use only the 16 basic terminal colors (well actually 256 since
you can theme 256 colors in kitty.conf). I dont think t_Co will work for
this. You need to use a color scheme in vim that is defined in terms of
terminal colors.

**Comment by page-down** [comment]
Add `colorscheme default` to the configuration file. Try this and see if it works for you.

```vim
if $TERM == "xterm-kitty"
    set termguicolors
    let &t_ut=''
    colorscheme default
endif
```

**Comment by wllclngn** [comment]
Awesome, thank you both! So, using @page-down 's code in *vimrc, save your colorscheme into "~/.vim/colors/[COLORSCHEME].vim".

Loving Kitty. Thanks!
